patches/tock/06-update-uicr.patch

```diff
@@ -0,0 +1,100 @@
+diff --git a/chips/nrf52/src/uicr.rs b/chips/nrf52/src/uicr.rs
+index 6bb6c86..3bb8b5a 100644
+--- a/chips/nrf52/src/uicr.rs
++++ b/chips/nrf52/src/uicr.rs
+@@ -1,38 +1,45 @@
+ //! User information configuration registers
+-//!
+-//! Minimal implementation to support activation of the reset button on
+-//! nRF52-DK.
```

Comment:
What changed w.r.t. this comment? Is the code now complete, or does it support more features (but not all of them) than the reset button?

Reply:
We already added support for voltage regulator and NFC pin protection.
This patch adds support for code read protection and the PR I'll prepare for Tock will add on top of that support for Nordic bootloader (2 words are stored in UICR and could be wiped, which could result in a bricked device).